Radar docs **DRAFT** (#3803)
redis/docs: content/operate/radar/manage-access.md

---
title: Manage access
alwaysopen: false
categories:
- docs
- operate
- radar
description: Create Radar users, assign roles, and connect LDAP or SAML SSO.
linkTitle: Manage access
weight: 40
---

> Radar has two roles, and every user has exactly one of them. People sign in with a local Radar account, through your directory over LDAP, or through your identity provider with SAML SSO.

Reviewer: ```suggestion
Radar has two roles, **admin** and **viewer**, one of which is assigned to every user. Users sign in with a local Radar account, through your directory over LDAP, or through your identity provider with SAML SSO.
```

Author: Addressed — added the role names: "administrator and viewer."

> Radar has two roles, and every user has exactly one of them. People sign in with a local Radar account, through your directory over LDAP, or through your identity provider with SAML SSO.

Reviewer 2: manage access is only relevant for on-prem deployments

Author: Addressed — covered by the page-level audience statement: "This page applies to self-managed Radar installations... roles... do not apply there."

> You configure users and sign-in methods in different places. Create local accounts under **Users**. Choose and configure the sign-in method under **Settings > Authentication**, where that choice applies to all of Radar.

Reviewer: Maybe needs a bit more expansion, but I don't think saying that you configure them in different places is necessary :-) 
```suggestion
Create local accounts under **Users**. Choose and configure the sign-in method under **Settings > Authentication**, where that choice applies to all of Radar.
```

Author: Addressed — applied verbatim.

## Roles

> | Role | Can do |

Reviewer 2: same with roles, etc. when you use cloud it's managed by the sm

Author: Addressed — covered by the page-level audience statement at the top: "Redis Cloud manages Radar users and roles for you... do not apply there."

|---|---|
| Admin | Everything a viewer can, plus add and remove connections, manage users, and change settings. |
| Viewer | Read the fleet: clusters, databases, usage, and alerts. |

A role applies to the entire fleet. You cannot scope a user to a subset of clusters.

New users are viewers unless you choose otherwise. Only admins can see or change users; the **Users** area is hidden from viewers.

## Create local users

> 1. Go to **Users** and add a user.

Reviewer: ```suggestion

To create a local user:

1. Go to **Users** and add a user.
```

Author: Fixed — dropped the filler line, goes straight to the numbered steps.

2. Enter a **username** and **email address**.
3. Set a **password**. It must be 6 to 128 characters and contain at least one uppercase letter.
4. Choose a **role**.

The **Users** list shows each account's role, where it authenticates, when it was created, and its last sign-in, so you can tell a local account from a directory account at a glance.

If an account locks, admins unlock it from the same list.

Radar creates no default account and ships no default password. The one-time bootstrap flow creates the first administrator while the database has no users. See [Install Radar]({{< relref "/operate/radar/install#5-create-the-first-administrator" >}}).

## Connect a directory with LDAP

Radar authenticates against your directory and takes each user's role from their group membership, so you grant and revoke Radar access by changing groups.

Configure LDAP under **Settings > Authentication**, in the **Active Directory / LDAP** section.

### Connect to the directory

> 

Reviewer: Sections with just a table as their content look a bit too clipped. If these sections correspond to tabs in the UI then the text should probably say something to that effect.

Author: Addressed — each LDAP/SAML subsection now has an intro sentence before its table.
